tool/tsh/db.go

```diff
@@ -767,6 +772,40 @@ Or view the connect command for the native database CLI client:
 		utils.Color(utils.Yellow, configCommand))
 }
 
+// printExtraConnectInfo prints extra instructions or information before
+// executing database client commands to connect to Teleport.
+func printExtraConnectInfo(cf *CLIConf, tc *client.TeleportClient, db *tlsca.RouteToDatabase, database types.Database) {
```

Comment:
I think it's a great idea to prompt the user that the user is not managed.

Though, I think my initial thought is to help people who just started using Teleport for Redis. It can be a habit that ppl input `auth` as the first thing once they are in. So this message just reminds them they do not have to do that.

Alternatively, I can display this when they actually input `auth <xxx> <xxx>` something, then db service can return a message that Teleport already logged you in?

Reply:
I think that there are two problems here:
1. I don't think it's a bad thing to show the message, but I also don't think that people will be confused without it. In most cases when you try to connect as a managed user someone needs to give you the username and password. If the user is managed technically no one can give you the password as only teleport knows it, so you won't try to use it.
2. What is the current behaviour when a user send `AUTH` command with incorrect password? I can imagine someone doing this even by mistake. If this somehow "break" the connection and a few minutes later "magically" fixes by itself when we rotate the password I'm thinking about always returning success on `AUTH` for managed users.

Reply:
yes. I will remove the tsh extra info for simplicity and catch if any managed users are sending `auth`. I'd love to return an `OK` but seems `redis-cli` hides any extra message sent with it. If we want to give extra information it has to be an error. I can play with that a bit.